chaos-test/Dockerfile

```diff
@@ -50,9 +50,9 @@ ENV LC_ALL=C.UTF-8
 ENV LANG=C.UTF-8
 
 ### Setup chaostoolkit packages
-ADD chaostest /app/chaos
-# ADD chaostoolkit-aws /app/chaostoolkit-aws
-# ADD chaostoolkit-kubernetes /app/chaostoolkit-kubernetes
+ADD chaostest /app/chaostest
```

Comment:
we can't make this change as this will require change in custom resource as well as charts ??